daffodil-lib/src/main/scala/org/apache/daffodil/lib/xml/DaffodilXMLLoader.scala

```diff
@@ -172,14 +174,25 @@ class DFDLCatalogResolver private ()
   }
 
   def resolveURI(uri: String): String = {
-    init
```

Comment:
For other reviewers: the reason why the `init` call is not needed at this line is because the next line calls `resolveCommon` which also calls `init` too.